include/init.h

```diff
@@ -26,6 +26,21 @@ extern "C" {
 #define _SYS_INIT_LEVEL_POST_KERNEL	2
 #define _SYS_INIT_LEVEL_APPLICATION	3
 
+extern s8_t z_sys_device_level;
+
+/**
+ * @brief Test whether startup has reached kernel main task.
+ *
+ * This impacts which services are available for use, and the context
+ * in which functions are run.
+ *
+ * @return true iff start up has completed all pre-kernel
```

Comment:
iff -> if

Unless, this means: if and only if ; in that case, pls, write full phrase :)